formdata-methods/index.html

```diff
@@ -0,0 +1,32 @@
+---
+feature_name: FormData methods for inspection and modification
+chrome_version: 50
+feature_id: 5701572055007232
+---
+
+<h3>Background</h3>
+<p>The <code>FormData</code> object helps to programatically send encoded form data to a server with an XHR. You can even generate it from a real <code>&lt;form&gt;</code>.</p>
+<p>These methods allow inspection and modification of the content of the <code>FormData</code>. Previously, this object could only be appended to, and never inspected. For a full list of methods, <a href="https://developer.mozilla.org/en/docs/Web/API/FormData" target="_blank">check MDN</a>.</p>
```

Comment:
Rather than link to MDN here, I'd just turn `FormData` in the first sentence into a link to MDN.